test/known_issues/test-vm-proxy-failure-CP.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const vm = require('vm');
+
+const handler = {
+    getOwnPropertyDescriptor: (target, prop) => {
+      throw new Error('whoops');
+    }
+};
+const sandbox = new Proxy({foo: 'bar'}, handler);
+const context = vm.createContext(sandbox);
+
+
+assert.throws(() => vm.runInContext('', context),
+        /Sandbox throws in CopyProperties() despite no code being run/);
```

Comment:
You're testing that the exception's error matches `Sandbox throws...`, which of course it doesn't because it is `whoops`. So the `assert` fails and it looks like the known issue passes. But we need to check that no exception is thrown, not that a made-up exception is not thrown. 

Why don't we change the assert to 
`assert.doesNotThrow(() => vm.runInContext('', context));` 
and add a comment in the beginning of the test: 
`// Sandbox throws in CopyProperties() despite no code being run.`